tests/utils/cookies.test.ts

```diff
@@ -138,6 +138,47 @@ describe('cookies', () => {
           ]);
         });
       });
+
+      describe('when NODE_ENV is not production', () => {
+        test('should not set a secure cookie on the response', async () => {
+          const { req, res } = getRequestResponse();
+          req.headers.host = 'www.acme.com';
+
+          process.env.NODE_ENV = 'development';
```

Comment:
This line of code is changing the environment variable and will potentially have side effects for other tests. Would it be possible to include some clean up logic, like this for example:

```js
  beforeEach(() => {
    initialEnvVar = process.env. NODE_ENV
  })
  afterEach(() => {
    process.env. NODE_ENV = initialEnvVar
  })
```

Reply:
(Sorry for the late followup)
Regarding NODE_ENV, this is already handled at the outer 'cookies' level:
```javascript

const originalEnv = process.env.NODE_ENV;
// ...
describe('cookies', () => {
  // ...
  afterEach(() => {
    timekeeper.reset();
    process.env.NODE_ENV = originalEnv;
  });
```
But you are right, I do add the `AUTH0_ALLOW_INSECURE_COOKIE` to `process.env` so should clean that up.